doc/source/dev/toolchain.rst

```diff
@@ -103,15 +103,16 @@ Official Builds
 Currently, SciPy wheels are being built as follows:
 
 ================  ========================  ===========================  ==============================
- Platform          Azure Base Image [14]_    Compilers                    Comment
+ Platform          Azure Base Image [5]_     Compilers                    Comment
 ================  ========================  ===========================  ==============================
-Linux (nightly)    ``ubuntu-18.04``          GCC 4.8                      See ``azure-pipelines.yml``
-Linux (release)    ``ubuntu-18.04``          GCC 7.5                      Built in separate repo [15]_
-OSX                ``macOS-10.14``           LLVM 11.0                    Built in separate repo [15]_
-Windows            ``VS2017-Win2016``        Visual Studio 2017 (15.9)    See ``azure-pipelines.yml``
+Linux (nightly)    ``ubuntu-18.04``          GCC 6.5                      See ``azure-pipelines.yml``
+Linux (release)    ``ubuntu-18.04``          GCC 7.5                      Built in separate repo [6]_
+OSX                ``macOS-10.15``           LLVM 12.0.0                  Built in separate repo [6]_
+Windows            ``windows-latest``        Visual Studio 2019 (16.11)   See ``azure-pipelines.yml``
 ================  ========================  ===========================  ==============================
 
-Note that the OSX wheels additionally vendor gfortran 4.8, see [15]_.
+Note that the OSX wheels additionally vendor gfortran 4.9,
+see submodule ``gfortran-install`` in [6]_.
```

Comment:
Just a note that we have to update this now that the minimum version of GCC is 6.3.0

Reply:
My understanding from the discussion [previously](https://github.com/scipy/scipy/pull/13347#issuecomment-971222849) was that the gfortran vendor is not directly linked to the GCC version used to compile scipy? In any case, I've been wanting to do this, but discussion of _where & how_ to repackage the conda-forge builds has [stalled](https://github.com/MacPython/gfortran-install/issues/7) a bit for now.